app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,32 @@
+class OrdersController < ApplicationController
+  def show
+    @order = Order.find_by(id: session[:order_id])
```

Comment:
While I'm sure this totally works as-is, there are some conventions that are pretty standard with rails regarding standard restful routes.

The customary bucket to pull out the id is from params.
When you use restful routes like you do when you use the `resources` method of declaring routes you actually get the id from the show route.

One way to see all the routes you have available is to type `rake routes` in your terminal.

Doing so you will see the one associated with show is `/orders/:id`

Any part of the route that has that colon will end up in params just by the act of going to that route. 

So `params[:id]` will give you the same value as `session[:order_id]` and is the more customary way to call it.